src/server/errors/ExternalComponentAlreadyTouchedError.ts

```diff
@@ -0,0 +1,10 @@
+import { IExternalComponent } from '../../definition/externalComponent';
+
+export class ExternalComponentAlreadyTouchedError implements Error {
+    public name: string = 'ExternalComponentAlreadyTouched';
+    public message: string;
+
+    constructor(externalComponent: IExternalComponent) {
+        this.message = `The app(${externalComponent.appId}) has already touched the external component(${externalComponent.name})`;
```

Comment:
```suggestion
        this.message = `The app (${externalComponent.appId}) has already touched the external component (${externalComponent.name})`;
```

Reply:
Thanks!

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/150/commits/f64e05be3c49c00693a37cdcf862d77cc6e11149

Reply:
Wonder if our lint shouldn't be hinting at this kind of problems 🤔 Will be checking

Reply:
That would be great to have a linter for stuff like this.